features/networking/service.feature

```diff
@@ -786,18 +786,18 @@ Feature: Service related networking scenarios
     Given I use the "<%= cb.masters[1].name %>" node
     #It should work because its external traffic from another node and destination node has a backend pod on it (ETP=local respected)
     When I run commands on the host:
-      | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> |
+      | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> || curl --connect-timeout 5 [<%= cb.hostip %>]:<%= cb.port %>|
     Then the step should succeed
     And the output should contain:
       | Hello OpenShift! |
     #It should NOT work because its external traffic from another node and destination node DOES NOT have a backend pod on it (ETP=local respected)
     When I run commands on the host:
-      | curl --connect-timeout 5 <%= cb.master0_ip %>:<%= cb.port %> |
+      | curl --connect-timeout 5 <%= cb.master0_ip %>:<%= cb.port %> || curl --connect-timeout 5 [<%= cb.master0_ip %>]:<%= cb.port %> |
```

Comment:
here cb.master0_ip  is still ipv4(it will be 127.0.0.1 in ipv6 single cluster), needs to get ipv6 in previous step, same as below.
```
 And the Internal IPv6 of node "<%= cb.masters[0].name %>" is stored in the :master0_ipv6 clipboard
```

Reply:
yes, 127.0.0.1 is not expected. 

I think we need another way to get the master hostip, maybe we can use api pod which scheduled on all masters and get the api_pod.hostip ?

Reply:
Ahh yea. I never noticed when we changed our internal ip node step def to differentiate between Ipv6 or Ipv4. We probably could have just gotten ```status.addresses[0]``` array value I believe which could have worked on both clusters probably. Looking at step def , I agree IPv6 needs to be fetched separately for master 0...Will try to play with it tomorrow and come up with solution.